packages/salesforcedx-apex-replay-debugger/src/core/logContext.ts

```diff
@@ -99,6 +113,98 @@ export class LogContext {
     );
   }
 
+  public getHeapDumps(): ApexHeapDump[] {
+    return this.apexHeapDumps;
+  }
+
+  public scanLogForHeapDumpLines(): boolean {
+    const heapDumpRegex = RegExp(/\|HEAP_DUMP\|/);
+    this.logLines.forEach((line, index) => {
+      if (heapDumpRegex.test(line)) {
+        const splitLine = line.split('|');
+        if (splitLine.length >= 7) {
+          const heapDump = new ApexHeapDump(
+            splitLine[3] /* heapDumpId */,
+            splitLine[4] /* className */,
+            splitLine[5] /* namespace */,
+            Number(splitLine[6]) /* line */
```

Comment:
This is more of a comment (no action required).

https://stackoverflow.com/questions/369220/why-should-you-not-use-number-as-a-constructor

This is OK since you are doing `Number(xx)` instead of `new Number(xx)`.

Reply:
@vazexqi FYI I was curious so I'd checked to see if it it would allow either and it doesn't. new Number(xx) would get flagged as incorrect syntax.